### PR TITLE
Add version control concept page and other fixes

### PR DESCRIPTION
**Pull request by hot007**
This PR addresses #38 adding a concept page on version control
#40 removing the ???s from the retire-replica page and making some minor formatting improvements, 
and updating the Table of Contents to include the version control page as well as addressing #56 and alphabetising the concept and some technical tip pages (per their titles in the rendered book), I hope!*
*I left backup strategy and backup checklist in that order because I think it makes more sense that way

Note this PR should not conflict with #55 so should be fine to review and merge them separately even though they're on the same branch. Note to @rlgregory if you do rely on any of these pages for your edits, you might need to hold off or 'sync' your fork once this PR is merged.

### CODE REVIEW

**Review by paolap — APPROVED** [review]
I only have 3 minor comments that can be dismissed or considered after merging:

1) for backup/backup-checklist in tech maybe we could move backup under concepts and keep the checklist in tech
2) version control looks good, my only comment is general is written as "we" other parts of the book might dress the user ("you) and other are impersonal. We should choose one at the next meeting and stick to it, or use different persons for different parts might be fine, but shouldn't be random
3) In retire-replica the **Note** could be rendered as a "note" box
```{note}
text in here
```

**Comment by paolap** [comment]
Thanks @hot007 great work!

**Comment by hot007** [comment]
Thanks for the comments @paolap they're good, I've addressed them in latest commit.

**Review by chloemackallah — APPROVED** [review]

File: Governance/retire/retire-replica.md [hunk 1]
```diff
@@ -19,24 +19,24 @@ Some methods are machine-driven, where you use any available statistics automati
...
+1. Use one or more automated method(s) to collect statistics on actual data usage.
 2. Announce to users with data access the anticipated retirement of the dataset. If the data appears to be unused, move to a "quarantine" area nearby where users may still find the data and notify the data managers that it is still required.
-3. If no objection is heard, the data can be staged for deletion. On the other hand, if objection is heard, then a dialogue must be started with the affected users to identify an alternative support mechanism for the dataset.
-    a. An alternative mechanism to save disk space while data is still required for infrequent access is to maintain the dataset on tape storage
+3. If no objection is heard, the data can be staged for deletion (moved to a "for deletion" location and read access to users removed). On the other hand, if objection is heard, then a dialogue must be started with the affected users to identify an alternative support mechanism for the dataset.</br>
+``` {retire}
+NOTE An alternative mechanism to save disk space while data is still required for infrequent access is to maintain the dataset on tape storage
+```
```
Comment (hot007) [inline Governance/retire/retire-replica.md:40]: @paolap this worked in preview but it looks like the "note" is not rendering in the final book?

Comment (hot007) [reply Governance/retire/retire-replica.md:40]: The note is not rendered and the numbering following is broken. Please feel free to maybe 'edit in production' if you can to fix it? maybe using a > blockquote instead?

**Pull request merged**